tests/ert/unit_tests/storage/test_local_storage.py

```diff
@@ -483,6 +483,60 @@ def fields(draw, egrid, num_fields=small_ints) -> List[Field]:
     ]
 
 
+@pytest.mark.usefixtures("use_tmpdir")
+@given(st.binary())
+def test_write_transaction(data):
+    with open_storage(".", "w") as storage:
+        filepath = Path("./file.txt")
+        storage._write_transaction(filepath, data)
+
+        assert filepath.read_bytes() == data
```

Comment:
deindent one level to make sure the context manager does not delete it (which I assume it is not meant to do)?

Reply:
ah, that context manager is probably irrelevant to _write_transaction.